library/src/main/java/bio/terra/landingzone/stairway/common/exception/MDCHandlingException.java

```diff
@@ -1,27 +0,0 @@
-package bio.terra.landingzone.stairway.common.exception;
```

Comment:
Is this not needed in general or it is in Stairway now too?

Reply:
Good question, it is not needed in general because Stairway doesn't need to serialize / deserialize MDC context like we had to do in our services when writing the calling thread's context to the flight input map.